_posts/2018-01-10-ats-garage-security-with-uptane.adoc

```diff
@@ -25,3 +25,12 @@ The second comes from the *Director*, which controls what updates (selected from
 {product-name-short} manages the Director for you. When you create an update campaign, we update the Director behind the scenes, signing the metadata for each image-device tuple in the campaign.
 
 The TUF keys should be managed on your side; we provide tooling to help you do just that. When you build a new device, bitbake automatically signs the image for you, using TUF keys that you specify in the build's `local.conf`{zwsp}footnote:[The TUF keys are packed inside the zip file specified by the `SOTA_PACKED_CREDENTIALS` line.]. Your initial key is created by {product-name}, delivered to you inside your `credentials.zip` file, and kept online for convenience and bootstrapping; for any production use you should link:../quickstarts/rotate-tuf-keys.html[rotate your TUF key] and store it securely offlinefootnote:[Proceed with caution! Once you take the key offline, HERE Technologies cannot recover it.].
+
+== Primary and secondary ECUs 
+
+anchor:primary-vs-secondary[]
```

Comment:
A minor nit-pick in three parts:

1. The anchor, if you're going to use one, ought to be attached to the section title, not the first paragraph
1. It would be more idiomatic in asciidoc to do it in one of these two ways:

```
== Primary and secondary ECUs [[primary-vs-secondary]]

[#primary-vs-secondary]
== Primary and secondary ECUs
```

However, you can also do it without even having to define an ID/anchor link--just use the slugified form: `link:../concepts/ats-garage-security-with-uptane.html#primary-and-secondary-ecus`